airflow/settings.py

```diff
@@ -169,6 +170,16 @@ def configure_orm(disable_connection_pool=False):
 configure_vars()
 configure_orm()
 
+# TODO: Unify airflow logging setups. Please see AIRFLOW-1457.
+logging_config_path = conf.get('core', 'logging_config_path')
+try:
+    from logging_config_path import LOGGING_CONFIG
+except Exception:
```

Comment:
Why have the catchall? I think if there's an exception we should just let it happen.

"Fail loudly"